project.Makefile

```diff
@@ -472,33 +472,33 @@ local/mongo_as_unvalidated_nmdc_database.yaml:
 		--page-size 200000 \
 		--schema-file src/schema/nmdc.yaml \
 		--selected-collections biosample_set \
-		--selected-collections data_object_set \
-		--selected-collections functional_annotation_agg \
-		--selected-collections study_set \
- 		--selected-collections extraction_set \
- 		--selected-collections field_research_site_set \
- 		--selected-collections library_preparation_set \
- 		--selected-collections mags_set \
- 		--selected-collections metabolomics_analysis_set \
- 		--selected-collections metagenome_annotation_set \
- 		--selected-collections metagenome_assembly_set \
- 		--selected-collections metagenome_sequencing_set  \
- 		--selected-collections metaproteomics_analysis_set \
- 		--selected-collections metatranscriptome_analysis_set \
- 		--selected-collections nom_analysis_set \
- 		--selected-collections omics_processing_set \
- 		--selected-collections pooling_set \
- 		--selected-collections processed_sample_set \
- 		--selected-collections read_based_taxonomy_analysis_set \
- 		--selected-collections read_qc_analysis_set
+		--selected-collections omics_processing_set \
+#		--selected-collections study_set \
+#		--selected-collections data_object_set \
+#		--selected-collections functional_annotation_agg \
+# 		--selected-collections extraction_set \
+# 		--selected-collections field_research_site_set \
+# 		--selected-collections library_preparation_set \
+# 		--selected-collections mags_set \
+# 		--selected-collections metabolomics_analysis_set \
+# 		--selected-collections metagenome_annotation_set \
+# 		--selected-collections metagenome_assembly_set \
+# 		--selected-collections metagenome_sequencing_set  \
+# 		--selected-collections metaproteomics_analysis_set \
+# 		--selected-collections metatranscriptome_analysis_set \
+# 		--selected-collections nom_analysis_set \
```

Comment:
I just noticed this list was originally in alphabetical order. @brynnz22, will you restore the original order and then comment in/out (or add/remove) the lines you want to enable/disable, while otherwise preserving the original order? I think that will simplify this diff and future diffs involving this file.

Reply:
@eecavanna I would, but I was running into issues where the commands I need to test the migration would only work if I put the exact collections I was working on at the top of the list. I'm not sure why this happens, but if I just uncommented them out in the middle, it would not grab the collections I needed.

Reply:
But if this doesn't matter for the migration, I can move them back to the preserved order? This was just for testing purposes.

Reply:
Ah, gotcha. That makes sense to me. The commented-out lines hide the line-continuation character from the shell, so the command looks like it ends at that line. I'll post a suggestion here in a minute (it will be to preserve the order of the ones that are not being modified).

Reply:
My last comment was in response to your previous one.

> I can move them back to the preserved order? 

That would work for me. I think you can revert these lines to their original state; and then leave the following change in place in this branch:

```diff
-		--migrator-name Migrator_from_9_1_to_9_2 \
+		--migrator-name Migrator_from_X_to_PR53 \
```

Reply:
Sounds good. I put them back in the original order that matches `main`

Reply:
Thanks! And I just realized they were mostly in the order I wanted already. I think it was just `study_set` that had been moved to the top unnecessarily and then commented out. I didn't realize everything under that was still in the original order already. Anyway, thanks for addressing this. This works for me.

Reply:
No problem. I think I edited the makefile on a couple of the other migration PRs. I will change those too.